projects/common/src/lib/utils/cds-utils.ts

```diff
@@ -15,4 +15,37 @@ export class CdsUtils {
     }
     return prefetch
   }
+
+  public static applySuggestionsUtil(result: any , serviceId: string) {
```

Comment:
Library utility function should not be depended on application implementation. Do not pass result (component) as input. Instead, make the function take suggestions, state and prefetch as input, return new prefetch after applying suggestions as output.

Reply:
Change the function name: applySuggestions~Utils~

Reply:
Revert the service name changes such as qriskService -> Service